custom_components/waste_collection_schedule/waste_collection_schedule/source/offenbach_de.py

```diff
@@ -32,7 +32,7 @@ def fetch(self):
 
         entries = self.fetch_year(now.year)
         if now.month == 12:
-            entries.append(self.fetch_year(now.year + 1))
+            entries = self.fetch_year(now.year + 1)
```

Comment:
Needs to be += or collections in deceber will not be shown in deceber.

Reply:
Totally right :D Thanks!